### ISSUE TITLE
Need help for training own datasets /求解训练过程中的一些疑问~

### ISSUE THREAD

**Issue by mokby** [bytedance/UNO#66]
Hi, thanks for your wonderful work! I am currently using your project to train my own datasets, but the training progress always stop at stpes 178, and the validation result is just pure noise, I don't know why this happended.

I did some changes as followed before training:
1.generated own datasets as the type you provided.
I created a json file that look like your `dreambench_multiip.json`, but I don't know where the put the target img,so after I read source code, my final json looks like this(179 pairs):
```json
[
  {
    "prompt": "xxxx",
    "images_paths": [
      "xxxx",
      "xxxx"
    ],
    "image_tgt_path": "xxx"
  },
]
````
as you see, I added `image_tgt_path` as target img, Is it correct?
2.I used the pre-trained lora you provide, and did some modify to load it using `resume_from_checkpoint`
main modifications in `trian.py`:
```python3
...
resume_from_checkpoint: str  = '/model/UNO'
...
def resume_from_checkpoint(resume_from_checkpoint,...):
....
path = resume_from_checkpoint
...
    lora_state = load_file(
        os.path.join(path, 'dit_lora.safetensors'),
        device=accelerator.device.__str__()
    )
....
    try:
        global_step = int(path.split("-")[1])
    except Exception as e:
        print(e)
        global_step = 0
```

3. There is no error after training stops, just some warnings below, I thought it might be the problem with wandb, so I use `wandb offline` to run offline, but not helpful:
```
Steps:   0%|                                                           | 178/100000 [30:21<225:28:18,  8.13s/it, lr=1e-5, step_loss=0.341]wandb:                                                                                
wandb: 
wandb: Run history:
wandb: train_loss ▁▆▂▂█▁▃▁▂▃▃▄▁▂▂▁▁▁▁▂▁▃▁▁▂▂▁▁▁▁▅▃▂▂▁▂▂▃▂▂
wandb: 
wandb: Run summary:
wandb: train_loss 0.34053
wandb: 
wandb: You can sync this run to the cloud by running:
wandb: wandb sync /home/llms/code/uno/wandb/offline-run-20250515_105406-lvdkbt4w
wandb: Find logs at: ./wandb/offline-run-20250515_105406-lvdkbt4w/logs
[RANK 0] 05/15/2025 11:25:11 - WARNING - urllib3.connectionpool - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)'))': /api/4504800232407040/envelope/
[RANK 0] 05/15/2025 11:25:11 - WARNING - urllib3.connectionpool - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)'))': /api/4504800232407040/envelope/
[RANK 0] 05/15/2025 11:25:11 - WARNING - urllib3.connectionpool - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)'))': /api/4504800232407040/envelope/
[RANK 0] 05/15/2025 11:25:12 - WARNING - urllib3.connectionpool - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)'))': /api/4504800232407040/envelope/
[RANK 0] 05/15/2025 11:25:12 - WARNING - urllib3.connectionpool - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)'))': /api/4504800232407040/envelope/
[RANK 0] 05/15/2025 11:25:13 - WARNING - urllib3.connectionpool - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)'))': /api/4504800232407040/envelope/
/opt/conda/envs/uno/lib/python3.10/site-packages/wandb/sdk/wandb_run.py:2382: UserWarning: Run (lvdkbt4w) is finished. The call to `_console_raw_callback` will be ignored. Please make sure that you are using an active run.
  lambda data: self._console_raw_callback("stderr", data),
```

**Comment by ValMystletainn** [comment]
1. the training data json is right.
2. also ok
3. how about just not init the wandb tracker and then check, to find out what if the breaking is cased by wandb or not. just commnet out the line 223-232 `accelerator.init_trackers(...)`

besides, to help debug the stop at step 178, can you check in different shuffle, the stop occurs or not. and also you should take the correction in issue #64

**Comment by mokby** [comment]
Many thanks, turns out my datasets is much too small. After I increased my datasets, promblem solved, again thanks for your wonderful work!
